lib/plistLib/main.js

```diff
@@ -127,7 +127,8 @@ define([
             return doc;
         },
         getType: function(value) {
-            if(value instanceof IntObject)
+            if(value instanceof IntObject
+                    || typeof value === 'number' && value === (value|0))
```

Comment:
I'm pretty sure this is the only way to interpret it. How can this not be in the spec?